github/repos_commits.go

```diff
@@ -27,6 +27,10 @@ type RepositoryCommit struct {
 	Stats *CommitStats `json:"stats,omitempty"`
 	// Details about which files, and how this commit touched. Only filled in during GetCommit!
 	Files []CommitFile `json:"files,omitempty"`
+
+	// The following fields are only populated by Webhook events.
```

Comment:
I think you can remove this "The following fields are only populated by Webhook events" disclaimer. Those fields should be included in other situations too, I think they were simply missing before.

See https://developer.github.com/v3/repos/commits/, it also includes `url` and `comments_url` in the repository commits.